dev/package.json

```diff
@@ -42,7 +42,7 @@
         "@types/mocha": "^7.0.2",
         "@types/node": "^16.0.0",
         "@types/terser-webpack-plugin": "^5.0.0",
-        "@types/vscode": "1.64.0",
+        "@types/vscode": "^1.77.0",
```

Comment:
Does 1.76 have the necessary types? I think most of our other packages have been bumped to that version at this point. Also, I'm not sure it actually is breaking, I think `LogOutputChannel` is just an interface so this would be strictly a dev-time import.

Reply:
1.76 does have the necessary types. 

And I thought it was a breaking change since updating to this version of the dev package will could create compile-time TS errors since the VS Code typings have breaking changes.